src/pages/gateway/headers.md

```diff
@@ -191,9 +191,13 @@ To add CORS headers to your mesh, create a `CORS` object in the `responseConfig`
 
 ## Retrieving handler details
 
-To receive response times and other additional details from a source, add `httpdetails=true` as a header value to your requests.
+To receive response times and other additional details from a source, add `httpDetails=true` as a header value to your requests.
```

Comment:
We removed support for this `httpDetails` request header and is being replaced with the new boolean field `includeHTTPDetails`